packages/material-ui/src/Tabs/Tabs.js

```diff
@@ -413,11 +541,10 @@ const Tabs = React.forwardRef(function Tabs(props, ref) {
   );
 
   const indicator = (
-    <TabIndicator
-      className={classes.indicator}
-      orientation={orientation}
-      color={indicatorColor}
+    <TabsIndicator
       {...TabIndicatorProps}
+      className={clsx(classes.indicator, TabIndicatorProps.className)}
+      styleProps={styleProps}
```

Comment:
Was the `color` prop intentionally moved to `stylesProps`?

Reply:
yes, `color` is for styling only. move to styleProps works the same.